google/resource_sql_database_instance.go

```diff
@@ -48,6 +48,7 @@ func resourceSqlDatabaseInstance() *schema.Resource {
 						"activation_policy": &schema.Schema{
 							Type:     schema.TypeString,
 							Optional: true,
+							Computed: true,
```

Comment:
A lot of the fields have defaults that come back from the API or have defaults that differ between first and second generation instances